src/pybind/mgr/dashboard/monitors.html

```diff
@@ -0,0 +1,106 @@
+
+{% extends "base.html" %}
+
+{% block content %}
+
+
+<script>
+        $(document).ready(function(){
+            // Pre-populated initial data at page load
+            var content_data = {{ content_data }};
+	     var refresh = function() {
+	         $.get("{{ url_prefix }}/monitors_data", function(data) {
+	             _.extend(content_data, data);
+	             $('.inlinesparkline').sparkline();
+	             setTimeout(refresh, 5000);
+	         });
+	     };	
+	     setTimeout(refresh, 5000);
+	
+	     rivets.formatters.sparkline_data = function(time_series) {
+	         result = "";
+	         for (var i = 1; i < time_series.length; ++i) {
+		     var y = time_series[i][1];
+		     result += (y + ",");		  
+	         }
+	         return result;
+	     };
+	
+             rivets.bind($("#content"), content_data);
+             $(".inlinesparkline").sparkline();
+         });
+</script>
+
+<!-- Page Header -->
+<section class="content-header">
+    <h1 style="font-weight:bold">
+      Monitors
+    </h1>
+</section>
+	
+<!-- Main content -->
+<section class="content text-center">
+    <div class="box">
+        <table class="table">
+            <tr style="font-size:15px">
+	         <td><p class="text-left"><span style="color:yellow; font-weight:bold">Cluster ID: </span> {mon_status.monmap.fsid}</p></td>
+	         <td><p class="text-left"><span style="color:yellow; font-weight:bold">Monmap Created: </span> {mon_status.monmap.created}</p></td>
+	         <td><p class="text-left"><span style="color:yellow; font-weight:bold">Monmap Modified: </span> {mon_status.monmap.modified}</p></td>
+	     </tr>
+        </table>	
+    </div>
+    <div class="box">
+        <h3 style="color:#00bb00; font-weight:bold">INSIDE QUORUM</h3>
+	 <div class="box-body">
+	     <table class="table table-bordered text-center">
+	         <thead>
+	             <th>Name</th>
+	  	     <th>Rank</th>
+		     <th>IP Addr</th>
+		     <th>Public Addr</th>
```

Comment:
`addr` and `public_addr` should be the same, and I'm pretty sure `addr` is only there for legacy reasons. In `MonMap::dump()`, both fields have the same value, and we should always honor `public_addr` in this case. What is missing though (and would be really useful) is showing the monitor's `public_bind_addr`, but I don't think we expose that on the monmap (it's a runtime thing). It should be exposed via  the daemon's `config get`, but I don't think that's reasonable to expect the dashboard to do, really.

In a nutshell, drop `IP Addr`.